system-addon/test/functional/mochitest/browser_topsites_section.js

```diff
@@ -54,3 +54,52 @@ test_newtab({
       "Topsite should be unpinned");
   }
 });
+
+// Check Topsites add
+test_newtab(
+  // it should be able to click the topsites edit button to reveal the edit topsites modal and overlay.
+  async function topsites_edit() {
+    // Find the edit topsites button
+    const topsitesEditBtn = content.document.querySelector(".edit-topsites-button button");
```

Comment:
The "Edit" is gone and replaced with "Add" now that skips the edit modal, so the intermediate step of clicking "Add" after "Edit" is not necessary anymore either.